charts/k8ssandra/templates/traefik.ingressroutes.yaml

```diff
@@ -43,3 +45,92 @@ spec:
         - name: {{ $clusterName }}-{{ $datacenterName }}-service
```

Comment:
`$clusterName` needs to be replaced with `$releaseName`.

Reply:
Oh, I've been using the same values for those in my testing. 🤦🏻‍♂️ Good catch, thanks.